File: test/ClientApp/src/app/__interfaces/assistance/AssistanceForRegistrationDto.ts
export interface AssistanceForRegistrationDto {
  IdUser?: string //set this by session/server side
  FechaInicialTurno: Date //serverside would be better
  FechaFinalTurno: Date //same that before
  IdSchedule?: number //set this on user or user array if multiple
  IdWorkplace: number //set this by user nearest location or attached to user depending on org
}

File: test/Entities/DTO/AssistanceForRegistrationDto.cs
using System;
using System.ComponentModel.DataAnnotations;
using test.Entities.Models;

namespace test.Entities.DTO
{
    public class AssistanceForRegistrationDto
    {
        public string? UserId { get; set; } // get this on the jwt not really necessary I think
        public DateTime FechaInicialTurno { get; set; } //set this on the server side
        public DateTime FechaFinalTurno { get; set; } //same as before
        public int? IdSchedule { get; set; } //if this belongs to a Employee then is attached to the obj itself
        public int IdWorkplace { get; set; } //same as before

    }
}

